### ISSUE TITLE
capital letters in the generated report (also in UI)

### ISSUE THREAD

**Issue by danielweck** [daisy/ace-gui#158]
Verbatim bug report from contributors / testers:

* In the table of the summary the titles wcag2a and wcag2aa should be written with capital letters and spaces.
* The index tabs (summary, violations, metadata, outlines, images) are presented in capital letters. For better readability, we suggest writing these in normal large and lower case letters.
